documentation/modules/ROOT/pages/configuration/outbox-event-router.adoc

```diff
@@ -221,9 +220,31 @@ apply the following configuration to the connector:
 ----
 transforms=outbox,...
 transforms.outbox.type=io.debezium.transforms.outbox.EventRouter
-transforms.outbox.table.fields.additional.placement=type:header:eventType
 value.converter=io.debezium.converters.ByteBufferConverter
 ----
 
-This moves the `eventType` value into a Kafka message header, leaving only the `payload` column value (the Avro data) as the sole message value.
-Using `ByteBufferConverter` as the value converter will propagate that value as-is into the Kafka message value.
+By default only the `payload` column value (the Avro data) is the sole message value.
+Using `ByteBufferConverter` as the value converter propagates that value as-is into the Kafka message value.
+
+=== Emitting messages with the event type field
+
+The `eventType` field is not emitted in the outbox routing SMT by default.
```

Comment:
See above, there shouldn't be any assumption about the `eventType` field. The contents in this section is good, though, it only needs slight tweaking to be presented as one example of using the additional fields mechanism: 

"Suppose in your outbox table there's a column "eventtype" which indicates the type a given event is of (e.g. for events of the "purchase order" aggregate type, there might be event types like "order created", "order shipped" etc.)..."